src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:33289/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:33289/project/issues?id=sample&pullRequest=6735&issues=AWk5i3mDLKQ6XO-7FdbC&open=AWk5i3mDLKQ6XO-7FdbC)

[ISSUE_KEY]: AWk5i3mDLKQ6XO-7FdbC